File: In C#/Pascal-Triangle.cs
using System;

class PascalTriangle
{
    static int Fact(int n)
    {
        int f = 1;
        for (int i = 1; i <= n; i++) f *= i;
        return f;
    }

    static int Ncr(int n, int r)
    {
        return Fact(n) / (Fact(r) * Fact(n - r));
    }

    static void Main()
    {
        Console.Write("Enter number of rows: ");
        int n = int.Parse(Console.ReadLine());

        for (int i = 0; i < n; i++)
        {
            Console.Write(new string(' ', n - i));
            for (int j = 0; j <= i; j++)
            {
                Console.Write(Ncr(i, j) + " ");
            }
            Console.WriteLine();
        }
    }
}

File: In Go/Pascal-Triangle.go
package main

import (
	"fmt"
)

func fact(n int) int {
	f := 1
	for i := 1; i <= n; i++ {
		f *= i
	}
	return f
}

func ncr(n, r int) int {
	return fact(n) / (fact(r) * fact(n - r))
}

func main() {
	var n int
	fmt.Print("Enter number of rows: ")
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		for j := 0; j < n - i; j++ {
			fmt.Print(" ")
		}
		for j := 0; j <= i; j++ {
			fmt.Printf("%d ", ncr(i, j))
		}
		fmt.Println()
	}
}
